apps/remix-ide-e2e/src/helpers/init.ts

```diff
@@ -5,7 +5,7 @@ require('dotenv').config()
 export default function (browser: NightwatchBrowser, callback: VoidFunction, url?: string, preloadPlugins = true): void {
   browser
     .url(url || 'http://127.0.0.1:8080')
-    .pause(5000)
+    .pause(10000)
```

Comment:
`init.ts` is used in all test files. A 5sec increase in delay will increase circleci time significantly.

Reply:
I have made this value 6000, I hope this is an acceptable value.